### PR TITLE
Added zip extractor folder

### PR DESCRIPTION
**Pull request by jayaganeshkumar**
# Pull Request Template

**script name** - zip_extractor

**What have you Changed**

what you changed in the codebase.write here

Created a folder with zip extractor python script along with requirements.txt and readme.md files

### Self Check(Tick After Making pull Request)

- [X] This issue was assigned to me.
- [x] One Change in one Pull Request
- [X] My file is in proper folder (Name of folder should be in lowercase with no space in between)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [X] I have added README.md and requirements.txt with my script

If issue was not assigned to you Please don't make a PR. It will marked as invalid.

### CODE REVIEW

**Review by pawangeek — CHANGES_REQUESTED** [review]
Correct requirements.txt

**Comment by jayaganeshkumar** [comment]
@pawangeek  what I have to do?

File: zip_extractor/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+zipfile37
```
Comment (pawangeek) [inline zip_extractor/requirements.txt:1]: Please correct this

Comment (jayaganeshkumar) [reply zip_extractor/requirements.txt:1]: It is the library name. Can I leave the library names as just "zipfile"

Comment (pawangeek) [reply zip_extractor/requirements.txt]: But what about the version. You can use "pip freeze > requiements.txt" to check that

Comment (jayaganeshkumar) [reply zip_extractor/requirements.txt]: I don't know that feature. If you know that can you update the readme file.

Comment (pawangeek) [reply zip_extractor/requirements.txt]: ok, that's an inbuilt library. My bad

**Review by pawangeek — APPROVED** [review]

**Comment by jayaganeshkumar** [comment]
Thank you so much @pawangeek

**Comment by jayaganeshkumar** [comment]
Can I have your linkedin profile id so that we can talk related to these kind of stuff?

**Comment by pawangeek** [comment]
You can use @pkjains to ping me on telegram. LinkedIn is not suitable for this kind of stuff

**Pull request merged**